applications/nublado/values.yaml

```diff
@@ -675,6 +675,60 @@ secrets:
   # user's lab.
   templateSecrets: true
 
+# If we're installing tutorials at a given site, this cronjob controls
+# their refresh.
+cronjob:
+  # -- Enable tutorials refresh?
+  enabled: false
+
+  # -- Source for repository to clone
+  gitSource: "https://github.com/lsst/tutorial-notebooks"
+
+  # -- Target where repository should land
+  gitTarget: "/project/cst_repos/tutorial-notebooks"
+
+  # -- Branch of repository to clone
+  gitBranch: "main"
+
+  # -- Where repository volume should be mounted
+  targetVolumePath: "/project"
```

Comment:
It seems that this may be redundant since the `targetVolume` defines a `mountPath` which gets used?